conversations/push-ios/didfinishlaunching/didfinishlaunching.m

```diff
@@ -0,0 +1 @@
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
```

Comment:
Many of these files are missing the trailing newline.

Reply:
Should all be fixed now